lib/tests/validation/validation.test.js

```diff
@@ -411,13 +411,13 @@ describe("Test getErrorList function", () => {
     const errorElement = document.createElement("div");
     errorElement.className = "gc-form-errors";
     document.body.appendChild(errorElement);
-    errorElement.focus = jest.fn();
+    const spyFocus = jest.spyOn(errorElement, 'focus');
     const spy = jest.spyOn(document, "getElementById").mockImplementation(() => {
       return errorElement;
     });
 
     setFocusOnErrorMessage(props, "gc-form-errors");
-    expect(errorElement.focus).toHaveBeenCalled();
+    expect(spyFocus).toHaveBeenCalled();
```

Comment:
I could not understand why this specific test suddenly stopped working. The error was that the `focus` function could not be assigned anything because it only implemented a `getter`.